frontend/app/src/router/routes.ts

```diff
@@ -163,6 +163,11 @@ export const Routes = {
     icon: 'mdi-chart-line',
     text: i18n.t('navigation_menu.manage_prices').toString()
   },
+  ETH_ADDRESS_BOOK_MANAGER: {
```

Comment:
btw as we migrate to vue3 we should start looking into [Vitesse](https://github.com/antfu/vitesse/) for improvement ideas. an thing that I liked for example was [vite-plugin-pages](https://github.com/hannoeru/vite-plugin-pages) that makes things closer to how they work with nuxt.